src/migrations/20230919103955-create-feature-environments-metrics-table.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS feature_environments_metrics(
```

Comment:
I am thinking that having foreign keys on feature_name and environment actually helps us. We could instead have something that cleans up unknown flags and environments every now and then. By dropping the constraints the insert operation would not need to validate against other tables.

This is also the approach we took for `client_metrics_env`.

Reply:
I also question the name. This is usage metrics and not feature toggle configurations. It is not an obvious difference between `client_metrics_env` and `feature_environments_metrics`

Reply:
This is a much better approach I agree.  I did think about it too - but was held back when I saw that we had tests checking exactly this: that we wouldn't store metrics for unknown features.

Reply:
dropping the constraints would solve most problems - we can still create an index on feature_name so we can easily query it when we need it